.vitepress/config.ts

```diff
@@ -95,6 +95,10 @@ export default defineConfig({
       '/platform/webcontainers/': sidebarLinks('main', ['webcontainers']),
       '/enterprise/': sidebarLinks('enterprise', ['enterprise']),
     },
+    chatlio: {
+      allowedRoutes: ["teams\/.*", "enterprise\/.*"],
```

Comment:
```suggestion
      allowedRoutes: ['/teams/.*', '/enterprise/.*'],
```

- No need to escape forward slashes in a string that gets passed to the `RegExp` constructor.
- We can start with a forward slash to avoid matching things like `'/other-category/about-teams/something'`. If we wanted to be even more restrictive, we'd need something like:

```ts
allowedRoutes: [
  `^${BASE}teams/.*`,
  `^${BASE}enterprise/.*`,
],
```

but that's probably overkill.

Reply:
actually the overkill looks good to me, let's be stricter for now.